rasa/core/tracker_store.py

```diff
@@ -364,8 +366,8 @@ def _set_key_prefix(self, key_prefix: Text) -> None:
     def _get_key_prefix(self) -> Text:
         return self.key_prefix
 
-    def save(self, tracker, timeout=None):
-        """Saves the current conversation state"""
+    def save(self, tracker: DialogueStateTracker, timeout: Any = None) -> None:
```

Comment:
the timeout has to be some sort of number, hasn't it? Maybe you can also look if we actually use the `timeout` parameter anywhere and potentially just remove it 🤷🏻 
```suggestion
    def save(self, tracker: DialogueStateTracker, timeout: Optional[float] = None) -> None:
```